xbmc/filesystem/VideoDatabaseDirectory/QueryParams.h

```diff
@@ -54,6 +55,7 @@ namespace XFILE
       long m_idAlbum;
       long m_idSet;
       long m_idTag;
+      long m_idVideoVersion;
```

Comment:
Please initialize the new member.

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#55-initialization

Reply:
All of its private members are initialized in the constructor, including this one, is it right to initialize only this one here? The guidelines seem to be for normal variables, not class members.

Reply:
> All of its private members are initialized in the constructor, including this one, is it right to initialize only this one here? The guidelines seem to be for normal variables, not class members.

Please default init the member. As said before, new code has to follow the code guidelines, not the style of the existing code surrounding the new code.

The code guidelines are clear on this imo: "Make sure that variables are initialized appropriately", as class members are ofc also variables and the guide is not referring to "local variables" but all kind of variables.